src/Inputs/Image.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState, useEffect, useCallback, useRef } from 'react';
+import 'react-image-crop/dist/ReactCrop.css';
+import ReactCrop from 'react-image-crop';
+import styled from 'styled-components';
+
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import { __useImplicitProps, ImplicitPropsInterface } from '@Utils/Hooks';
+import { transition, position, flex } from '../Utils/Mixins';
+import { Heading } from '../Text/Heading';
+import { Button } from './Button';
+import { Modal } from '../Containers/Modal';
+
+export interface ImageProps
+    extends MainInterface,
+        ResponsiveInterface,
+        ImplicitPropsInterface {
+    maxWidth?: number | undefined;
+    accept?: string;
+    aspect?: number;
+    onChange?: Function;
+}
+
+const Drop = styled.input`
+    ${position('absolute', 'auto', 0)}
+    opacity: 0;
+    cursor: pointer;
+    height: 100%;
+    width: 100%;
+`;
+
+const Container = styled.div<ImageProps>`
```

Comment:
These styles are very outdated, perhaps we should make this a button

Reply:
Make the Container component a button instead of a div?